lib/gc/Transforms/Pipeline.cpp

```diff
@@ -32,6 +32,17 @@
 
 namespace mlir::gc {
 
+void populateCleanUpPasses(mlir::OpPassManager &pm) {
```

Comment:
Why do we need LICM/mem2reg/topologicalSort pass inside cleanup stage?

Reply:
Has moved the LICM/mem2reg out of clean up stage and only call them when needed